Gems/EMotionFX/Code/Source/Integration/Components/ActorComponent.h

```diff
@@ -94,6 +94,8 @@ namespace EMotionFX
                 bool m_forceUpdateJointsOOV = false;
                 BoundingBoxConfiguration m_bboxConfig; ///< Configuration for bounding box type and updates
 
+                bool m_excludeFromReflectionCubeMaps = false;
```

Comment:
```suggestion
                bool m_excludeFromReflectionCubeMaps = true;
```
From the discussion it sounds like we want to default this to exclude actors from reflections cube maps by default.